src/pages/waiting-room-to-car/waiting-room-to-car.ts

```diff
@@ -100,7 +109,7 @@ export class WaitingRoomToCarPage extends BasePageComponent {
 
   @ViewChild('instructorRegistrationInput')
   instructorRegistrationInput: ElementRef;
-  inputSubscriptions: Subscription[] = [];
+  // inputSubscriptions: Subscription[] = [];
```

Comment:
Can this be removed now input subscriptions are no longer used?

Reply:
Removed this - also removed dispatching action function that is no longer necessary